.github/workflows/internal-ci.yaml

```diff
@@ -50,23 +50,44 @@ jobs:
       dry-run: true
       version: ${{ needs.test_version.outputs.version }}
 
+  get_image_extended_test:
```

Comment:
i think this is nice, by default they don't have to specify anything apart from the image name but everything is overridable and they can use the p2p-get-latest-image directly if needed